browser/ui/sidebar/sidebar.h

```diff
@@ -19,6 +19,9 @@ class Sidebar {
   // Update sidebar item's UI state.
   virtual void UpdateSidebarItemsState() = 0;
 
+  // Return true if there is active contextual panel.
+  virtual bool HasActiveContextualEntry() = 0;
```

Comment:
I'd be great if we have what "contextual panle" means. Like, "associated with web contents in tab strip model from upstream.".

Reply:
Updated.